2D Simulation/game.js

```diff
@@ -122,10 +122,14 @@ socket.on('enable_blocks_for_player_2', function(data) {
 });
 
 socket.on('update_position', function (moveData) {
+    update_position(moveData);
+});
```

Comment:
If you're going to pull it out in a function, it's better to use `socket.on('update_position', update_position)` as an eta reduction.